dotnet/src/webdriver/Internal/Logging/FileLogHandler.cs

```diff
@@ -76,9 +80,14 @@ public FileLogHandler(string filePath, bool overwrite)
         /// <param name="logEvent">The log event to handle.</param>
         public void Handle(LogEvent logEvent)
         {
+            if (_streamWriter is not { } writer)
```

Comment:
Using this class after it's disposed currently throws a `NullReferenceException`. It's not too important what happens when you use a disposed object, but a "best-effort" attempt to throw `ObjectDisposedException` is always nice. Let me know if this is unnecessary.

Reply:
Not in this case, internal logging should be always be safe - no runtime exceptions at all. It is not a criminal that somebody disposed writer and continues to write (in our case), we should be silent to not interrupt more important execution flow - users code. This is current understanding when we introduced internal logging mechanism. Let's keep it for a while.

Reply:
Internal logging was kind of one time commit, I am happy that it works and it doesn't interrupt user's execution. Any change here are better proceeded as separate PR, considering we know bugs.

Reply:
That makes sense to me. This change has been reverted.